src/goto-instrument/cover.h

```diff
@@ -31,7 +31,11 @@ class optionst;
 
 #define HELP_COVER                                                             \
   " --cover CC                   create test-suite with coverage criterion "   \
-  "CC\n"                                                                       \
+  "CC,\n"                                                                      \
+  "                              where CC is one of assertion, assume,\n"      \
+  "                              branch, condition, cover, decision,\n"        \
+  "                              location, mcdc, or path\n"                    \
+  "                              (or the plural of these keywords)\n"          \
```

Comment:
except for `mcdc` and `cover`... I'd drop the mention of plural.
Has `path` ever been implemented?

Reply:
Thanks, I've updated the text. Also, I thus removed "path" as it seems the only "implementation" is a "TODO" ...